TLM/TLM/Persistence/IPersistentObject.cs

```diff
@@ -0,0 +1,22 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Xml.Linq;
+
+namespace TrafficManager.Persistence {
+    internal interface IPersistentObject : IComparable, IComparable<IPersistentObject> {
```

Comment:
Yes, I will do this. (It's not copy/pasted code, it's all new.)

I just felt a little urgency to get people's eyes on this, because @krzychu124 is waiting on it for Airports DLC enhancements.

btw, I will probably be cleaning up this framework a little more and publishing it separately as an open-source library, which I'll then use in other mod work. Whether TMPE switches over to that or continues to use its own version, is a decision that can be made when the time comes.